examples/convert_mri_and_trans.py

```diff
@@ -155,10 +156,26 @@
               title=label)
 plt.show()
 
+###############################################################################
+# We can write another types of MRI data such as FLASH images for BEM models
+
+flash_mgh_fname = \
+    op.join(data_path, 'subjects', 'sample', 'mri', 'flash', 'mef05.mgz')
+
+flash_bids_path = \
+    BIDSPath(subject=sub, session=ses, root=output_path, suffix='FLASH')
+
+write_anat(
+    image=flash_mgh_fname,  # path to the MRI scan
+    bids_path=flash_bids_path,
+    raw=raw,  # the raw MEG data file connected to the MRI
+    verbose=True  # this will print out the sidecar file
```

Comment:
```suggestion
    image=flash_mgh_fname,
    bids_path=flash_bids_path,
    raw=raw,
    verbose=True
```

We have the comments in the code snippet above already, so I think it's save to omit them here :)